crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -313,19 +369,28 @@ impl TryFrom<&types::PaymentsCancelRouterData> for CancelOrCaptureTransactionReq
     }
 }
 
-impl TryFrom<&types::PaymentsCaptureRouterData> for CancelOrCaptureTransactionRequest {
+impl TryFrom<&AuthorizedotnetRouterData<&types::PaymentsCaptureRouterData>>
+    for CancelOrCaptureTransactionRequest
+{
     type Error = error_stack::Report<errors::ConnectorError>;
-    fn try_from(item: &types::PaymentsCaptureRouterData) -> Result<Self, Self::Error> {
+    fn try_from(
+        item: &AuthorizedotnetRouterData<&types::PaymentsCaptureRouterData>,
+    ) -> Result<Self, Self::Error> {
         let transaction_request = TransactionVoidOrCaptureRequest {
             amount: Some(utils::to_currency_base_unit_asf64(
-                item.request.amount_to_capture,
-                item.request.currency,
+                item.router_data.request.amount_to_capture,
```

Comment:
amount can be fetched directly from AuthorizedotnetRouterData as conversion is already happening in `AuthorizedotnetRouterData` tryfrom
```suggestion
                item.amount.clone(),
```

Reply:
amount in TransactionVoidOrCaptureRequest is a f64 and amount in AuthorizedotnetRouterData is a String